packages/consent-manager/index.js

```diff
@@ -59,14 +62,9 @@ export default class ConsentManager extends Component {
         this.props.additionalServices
       )
       return false
-    }
-    // 2. Check inEU. If in-eu, show consent. If not, show nothing. Load all.
-    if (inEU() || this.props.forceShow) {
-      return true
     } else {
-      // Load all analytics
-      this.saveAndLoadAnalytics({ loadAll: true })
```

Comment:
Just want to make sure removing this still would make it work?

Reply:
we're good here - this line was the "not in the EU, so load everything by default" which we no longer want to do. Thanks for flagging for a double check here.